### PR TITLE
Refactor plain text block

### PR DESCRIPTION
**Pull request by gustaveen**
Plain text block needed for the about us page

Screenshot 2020-05-22 at 14 52 41

### CODE REVIEW

File: src/blocks/PlainTextBlock.tsx [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
 
-const PlainTextSectionWrapper = styled(SectionWrapper)({
-  paddingBottom: '0 !important',
-})
+type ParagraphFontSizes = 'sm' | 'md' | 'lg' | 'xl'
 
-const TitleWrapper = styled('div')({
-  paddingBottom: '2.5rem',
-})
+const paragraphSizeMap = {
```
Comment (palmenhq) [inline src/blocks/PlainTextBlock.tsx:15]: Just an idea - what about specifying this as `Record<ParagraphFontSize, string>`?

File: src/blocks/PlainTextBlock.tsx [hunk 2]
```diff
@@ -1,54 +1,57 @@
...
+} from '../components/blockHelpers'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+} from './BaseBlockProps'
 
-const PlainTextSectionWrapper = styled(SectionWrapper)({
-  paddingBottom: '0 !important',
-})
+type ParagraphFontSizes = 'sm' | 'md' | 'lg' | 'xl'
```
Comment (palmenhq) [inline src/blocks/PlainTextBlock.tsx:13]: imo this should be in singular, as you'll only use one at a time. `ParagraphFontSize`

File: src/blocks/PlainTextBlock.tsx [hunk 3]
```diff
@@ -1,54 +1,57 @@
...
     extraStyling={extra_styling}
   >
-    <ContentWrapper index={index}>
-      <TitleWrapper>
-        <h1>{title}</h1>
-      </TitleWrapper>
+    <ContentWrapper brandPivot index={index}>
+      <InnerContent
+        fontSize={font_size}
+        dangerouslySetInnerHTML={{ __html: content && content.html }}
```
Comment (palmenhq) [inline src/blocks/PlainTextBlock.tsx:53]: `content?.html`?

File: src/blocks/PlainTextBlock.tsx (hunk 1 above)
Comment (gustaveen) [reply src/blocks/PlainTextBlock.tsx]: Nice!

**Comment by gustaveen** [comment]
Thanks, updated now :) Amended the wrong commit though haha 🙃

**Review by palmenhq — APPROVED** [review]
awesome!

**Pull request merged**